lib/index.d.ts

```diff
@@ -100,6 +110,8 @@ declare namespace Realm {
         [keys: string]: any;
     }
 
+    type ObjectChangeCallback = (object: Object, changes: any) => void;
```

Comment:
Shouldn't we create an `ObjectChangeSet` interface here instead of `any` ?

Reply:
true